voyager-publish/index.js

```diff
@@ -9,28 +9,59 @@ export const publish = async ({
   maxMeasurements = 1000,
   logger = console
 }) => {
+  // Lock measurements for this run
+  // - On success, measurements will be deleted
+  // - On failure, lock will be released after 5 minutes
```

Comment:
I don't see any code to release the lock, is this something that Postgres perform automatically?

Reply:
I'm surprised, I must have lost that line. Added it back